test/provision/k8s_install.sh

```diff
@@ -20,7 +20,11 @@ source ${PROVISIONSRC}/helpers.bash
 # TODO: Check if the k8s version is the same
 
 if [[ -f  "/etc/provision_finished" ]]; then
-    sudo dpkg -l | grep kubelet
+    if grep Ubuntu /etc/lsb-release; then
```

Comment:
I think that this code can be deleted no?

Reply:
I don't know. It can be? I didn't introduce anything else there.